test/sql/size_utils.sql

```diff
@@ -56,4 +62,4 @@ SELECT * FROM hypertable_approximate_row_count(NULL);
 
 SELECT * FROM chunks_detailed_size(NULL);
 SELECT * FROM hypertable_detailed_size(NULL);
-SELECT * FROM indexes_relation_size(NULL);
+SELECT * FROM hypertable_index_relation_size(NULL);
```

Comment:
I don't see any negative tests, e.g., what happens if you pass in a relation name or an Oid that is incorrect?

Reply:
test aded